entrypoint.sh

```diff
@@ -5,7 +5,7 @@ case "$1" in
     develop)
         echo "Running Development Server"
         echo -e "$GCLOUD_STORAGE" | base64 -d > storage.json
-        exec grunt --gruntfile app/Gruntfile.js | bunyan
+        exec yarn start | bunyan
```

Comment:
this is unrelated... not sure what impact it may have

Reply:
This gets a node server running. The grunt command is throwing an error which will have to be tackled in a different PR. The downside is that this doesn't have live reload.. something that can be addressed with the proper tool when these MSs are updated.